### PR TITLE
WL-3506: Don't show page links on Preview tab

### PR DESCRIPTION
**Pull request by nicholaswilson100**
Also remove an unused script (get-html.js) and fix a bug.

### CODE REVIEW

File: library/src/webapp/editor/ckextraplugins/oxam/js/oxam-embed.js [hunk 1]
```diff
@@ -36,8 +36,10 @@ $.fn.oxamEmbed = function(options) {
     var params = {};
 
     for (i in tmpParams) {
-      var param = tmpParams[i].split('=');
-      params[param[0]] = param[1];
+      if (i!='contains'){
```
Comment (buckett) [inline library/src/webapp/editor/ckextraplugins/oxam/js/oxam-embed.js]: Why are you checking for contains?

**Comment by nicholaswilson100** [comment]
'contains' is in this iterator array (it shouldn't be).  This is the same quick fix to get this widget working as a couple of other places (folder-listing.js (line 35), oxam-embed (line 39).

File: library/src/webapp/editor/ckextraplugins/oxam/js/oxam-embed.js (hunk 1 above)
Comment (buckett) [inline library/src/webapp/editor/ckextraplugins/oxam/js/oxam-embed.js]: The better fix is normally to switch to a sequential loop: http://stackoverflow.com/questions/3010840/loop-through-array-in-javascript

Comment (nicholaswilson100) [inline library/src/webapp/editor/ckextraplugins/oxam/js/oxam-embed.js]: Cool!, I'll do that.

Comment (nicholaswilson100) [inline library/src/webapp/editor/ckextraplugins/oxam/js/oxam-embed.js]: Thanks

**Comment by nicholaswilson100** [comment]
Thanks

**Pull request merged**